lib/webview-helpers.js

```diff
@@ -80,29 +81,60 @@ async function procFromWebview (adb, webview) {
   let out = await adb.shell("ps");
   let pkg = "unknown";
   let lines = out.split(/\r?\n/);
+
   /* Output of ps is like:
-   USER     PID   PPID  VSIZE  RSS     WCHAN    PC         NAME
+   USER       PID  PPID  VSIZE  RSS   WCHAN    PC         NAME  _or_
+   USER       PID  PPID  VSZ    RSS   WCHAN    ADDR     S NAME
    u0_a136   6248  179   946000 48144 ffffffff 4005903e R com.example.test
+   u0_a136   6249  179   946000 48144 ffffffff          R com.example.test
   */
-  let header = lines[0].trim().split(/\s+/);
+  let header = lines[0];
+
   // the column order may not be identical on all androids
   // dynamically locate the pid and name column.
-  let pidColumn = header.indexOf("PID");
-  let pkgColumn = header.indexOf("NAME") + 1;
+  let pidIndex = header.indexOf('PID');
+  let pkgIndex = header.indexOf('NAME');
+
+  function slice (line, index) {
```

Comment:
i eventually figured out what this function does, but a little comment describing what its purpose is would be helpful.

Reply:
Fixed!